src/foam/u2/filter/DateFilterView.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.filter',
+  name: 'DateFilterView',
+  extends: 'foam.u2.Controller',
+
+  documentation: `A SearchView for properties of type Date.`,
+
+  requires: [
+    'foam.mlang.predicate.True',
+  ],
+
+  properties: [
+    {
+      name: 'property',
+      documentation: `The property that this view is filtering by. Should be of
+          type Date.`,
+      required: true
+    },
+    {
+      class: 'String',
+      name: 'qualifier',
+      documentation: `Lets the user choose a predicate to filter the view by.`,
+      view: {
+        class: 'foam.u2.view.ChoiceView',
+        choices: [
+          ['True', 'All Time'],
+          ['Gt', 'After'],
+          ['Lt', 'Before'],
+          ['Bt', 'Between']
+        ],
+        defaultValue: 'True'
+      }
+    },
+    {
+      class: 'Date',
+      name: 'date1',
+      documentation: `A date selectable by the user. Used as the second argument
+          to the predicate generated by this view.`,
+    },
+    {
+      class: 'Date',
+      name: 'date2',
+      documentation: `A date selectable by the user. Used as the second argument
+          to the predicate generated by this view.`,
+    },
+    {
+      name: 'predicate',
+      documentation: `All SearchViews must have a predicate as required by the
+          SearchManager. The SearchManager will read this predicate and use it
+          to filter the dao being displayed in the view.`,
+      expression: function(qualifier, date1, date2) {
+        if ( ! qualifier || ! date || ! isNaN(date.valueOf()) ) return this.True.create();
```

Comment:
If instead of importing True you implement: foam.mlang.Expressions, then you can just do this.TRUE.

Reply:
true. I was just doing a very mcarcaso thing to do. Haha.